sympy/series/limits.py

```diff
@@ -146,15 +146,14 @@ def doit(self, **hints):
 
         if e.is_Mul:
             if abs(z0) is S.Infinity:
-                # XXX todo: this should probably be stated in the
-                # negative -- i.e. to exclude expressions that should
-                # not be handled this way but I'm not sure what that
-                # condition is; when ok is True it means that the leading
-                # term approach is going to succeed (hopefully)
+                ztemp = Symbol("z",real = True)
```

Comment:
It is not necessary to change the variable to find the argument of an exponential.
```
>>> m = exp(x**2 - 1)
>>> m.args[0]
x**2 - 1
```